src/Table.hxx

```diff
@@ -118,6 +121,9 @@ public:
     void write_ipac_table(const boost::filesystem::path &p);
 
     std::vector<size_t> get_column_widths() const;
+    // G2P calls this function, so can't simply rename it.  :-(
+    std::vector<size_t> get_column_width() const { return get_column_widths(); }
```

Comment:
Since we're using C++14, how about marking this [deprecated](https://en.cppreference.com/w/cpp/language/attributes/deprecated)?

Reply:
Nice, didn't know about that.